scipy/linalg/decomp_svd.py

```diff
@@ -405,7 +405,7 @@ def subspace_angles(A, B):
     Returns
     -------
     angles : ndarray, shape (min(N, K),)
-        The subspace angles between the column spaces of `A` and `B`.
+        The subspace angles between the column spaces of `A` and `B` in ascending order.
```

Comment:
We should probably make this descending order so there is no backward compat break